### ISSUE TITLE
Ignoring certain characters in linux

### ISSUE THREAD

**Issue by cancub** [repeats/Repeat#6]
Sup, HP?
Using Ubuntu 16.04
I've noticed a small bug when I set up a host removal hotkey like so:
```
k.type(VK_BACK_SPACE);
k.type(VK_BACK_SPACE);
k.type(VK_BACK_SPACE);
k.type("ssh-keygen -f \"/home/<user>/.ssh/known_hosts\" -R <ip>"); 
```
For some reason, the "w" in "known" is dropped. Result from the call to the activation key combo of Q+W+NumPad-1:

    ssh-keygen -f "/home/<user>/.ssh/knon_hosts" -R <ip>

However, adding a second "w" (i.e., "knowwn") makes everything fine.

    ssh-keygen -f "/home/<user>/.ssh/known_hosts" -R <ip>

It seems like this has to do with the keys used for the combo as they appear to be the letters that are being removed from the output. Using "Y+H+K" results in

    ss-egen -f "/home/<user>/.ssh/known_hosts" -R <ip>

Finally, the error occurs with the key on the keyboard itself, not the character created. Using "S+2" for 

    ssh root@<IP>

will result in 

    sh root<ip>

implying that either 2 or @  (same keyboard key) will be removed by the 2 in S+2

**Comment by hptruong93** [comment]
TLDR: Try enabling "Execute on release" option under Settings menu. 

Take an example of typing "ase" using the key chain Q + W + E, at the time the last button of the hotkey chain is pressed down (i.e. the button E is pressed), the task is activated and your code instructs it to execute the typing commands, which happens by simulating a key press followed by a key release for each letter.
The undesired behavior is caused by the fact that at the time the simulated key press followed by the simulated key release for button E is registered, you have not finished lifting your hand from the physical button E on the keyboard. This results in the OS probably receiving PRESS (real), PRESS (simulated), RELEASE (simulated), RELEASE (real) signal on button E, which caused the undesired behavior.

We can resolve this by configuring Repeat to execute the compiled task AFTER the hotkey chain has been completely released (i.e. after you lifted you finger from the button E on the keyboard).

**Comment by cancub** [comment]
Yup, that looks to have solved the issue. Thanks for your help.

**Comment by cancub** [comment]
looks like this issue is still around

**Comment by hptruong93** [comment]
Can you provide a reproducible example? I've never had problem after using to "Execute on release" option.

**Comment by cancub** [comment]
ahhh, it was disabled after re-installation and I completely forgot that it was an option. durr

**Comment by hptruong93** [comment]
If you had use the in place binary replacement, all your configurations would have been there too. :)
